src/ClashOfClans.App/ClashOfClans.App.csproj

```diff
@@ -7,10 +7,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ClashOfClans" Version="3.0.5" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\ClashOfClans\ClashOfClans.csproj" />
```

Comment:
This is an example app of how to use ClashOfClans NuGet library and it's API, kind of "template" and thus it should use NuGet packet instead of direct project reference.